lua/neogen/configurations/julia.lua

```diff
@@ -0,0 +1,171 @@
+local ts_utils = require("nvim-treesitter.ts_utils")
+local nodes_utils = require("neogen.utilities.nodes")
+local extractors = require("neogen.utilities.extractors")
+local locator = require("neogen.locators.default")
+local template = require("neogen.template")
+local i = require("neogen.types.template").item
+
+local parent = {
+    func = { "function_definition" },
+    -- struct = { "struct_definition" },
+}
+
+
+return {
+    -- Search for these nodes
+    parent = parent,
+
+    -- Traverse down these nodes and extract the information as necessary
+    data = {
+        func = {
+            ["function_definition"] = {
+                ["0"] = {
+                    extract = function(node)
+                        local results = {}
+
+                        local tree = {
+                            {
+                                retrieve = "all",
+                                node_type = "parameter_list",
+                                subtree = {
+                                    { retrieve = "all", node_type = "identifier", extract = true },
+                                    {
+                                        retrieve = "all",
+                                        node_type = "typed_parameter",
+                                        subtree = { { retrieve = "all", node_type = "identifier", extract = true } },
```

Comment:
Maybe try with 

```lua
subtree = {
  {
    position = 1,
    extract = true,
    as = I.Identifier
  },
    {
    position = 2,
    extract = true,
    as = ...
  },
}
```

Reply:
if this doesn't work, just retrieve the parent node aka `typed_parameter` and do a Lua separation with `::`.